django/core/validators.py

```diff
@@ -397,8 +397,12 @@ class StepValueValidator(BaseValidator):
     message = _("Ensure this value is a multiple of step size %(limit_value)s.")
     code = "step_size"
 
+    def __init__(self, limit_value, offset=None, message=None):
+        super().__init__(limit_value, message)
+        self.offset = 0 if offset is None else offset
```

Comment:
If 0 goes in the sig:

```suggestion
        self.offset = offset
```